pkg/gcv/result.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcv
+
+import (
+	"encoding/json"
+
+	"github.com/forseti-security/config-validator/pkg/api/validator"
+	"github.com/golang/protobuf/jsonpb"
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	cftypes "github.com/open-policy-agent/frameworks/constraint/pkg/types"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// Result is the result of reviewing an individual resource
+type Result struct {
+	// The name of the resource as given by CAI
+	Name string
+	// CAIResource is the resource as given by CAI
+	CAIResource map[string]interface{}
+	// ReviewResource is the resource sent to Constraint Framework for review.
+	// For GCP types this is the CAI object, for K8S types, this is the unwrapped
```

Comment:
does this mean for GCP type, CAIResource == ReviewResource?

Reply:
Yes, that's correct for GCP it will match, but for K8S stuff in CAI, this will be the unwrapped resource.